0_reporting_functions.R

```diff
@@ -377,27 +384,28 @@ ReportGeneration <- function(){
   portfolio_name_selectClean <- gsub("'","",portfolio_name_selectClean)
   
   
-  text$text <- gsub("start_year+10",start_year+10,text$text,fixed=T)
-  text$text <- gsub("start_year+5",start_year+5,text$text,fixed=T)
-  text$text <- gsub("start_year",start_year,text$text)
-  text$text <- gsub("investor_name_select",investor_name_selectClean,text$text)
-  text$text <- gsub("portfolio_name_select",portfolio_name_selectClean,text$text)
+  text$text <- gsub("Startyear+10",start_year+10,text$text,fixed=T)
+  text$text <- gsub("Startyear+5",start_year+5,text$text,fixed=T)
+  text$text <- gsub("Startyear+5",start_year+5,text$text,fixed=T)
+  text$text <- gsub("Startyear",start_year,text$text)
+  text$text <- gsub("InvestorName",investor_name_selectClean,text$text)
+  text$text <- gsub("PortfolioName",portfolio_name_selectClean,text$text)
```

Comment:
this looks a bit like the old naming convention is reintroduced? should that be changed?

Reply:
Nay, unfortunately the old naming convention is still present in the Template .tex files (afaik). 
Maybe open an issue to align the Template files to the new convention? and then we can change these back.

Reply:
Here for example: https://github.com/2DegreesInvesting/Templates/blob/77db076c87b5a136854d8a1a3d417ded1a3215fa/GeneralTemplateInput_v6_SB_en.tex#L255

Actually not sure if it's worth fixing, because I think we need to be moving away from this type of Report generation altogether, and towards bookdown (or gitbook as people erroneously call it hehe)

Reply:
I mean, as long as we have two computers that still have a functioning version of the Report generation.. maybe we can decide this together with @Clare2D on Monday

Reply:
Yep as Jackson suggests, I don't think this is worth changing. If we want to use a different template, we'd need to go through all templates to fix this. Hopefully, we move on soon!